src/app/grid/grid-external-advanced-filtering/grid-external-advanced-filtering.component.scss

```diff
@@ -1,10 +1,12 @@
 .grid__wrapper {
     display: flex;
-    flex-flow: column;
+    flex-direction: column;
     margin: 15px;
+    align-items: stretch;
 }
 
 .advanced-dialog {
     margin-bottom: 2px;
-    height: 300px;
-}
\ No newline at end of file
+    max-height: 325px;
```

Comment:
```suggestion
    height: 325px;
```

Reply:
This way the grid will not jump up and down when adding filtering expressions.
![grid-filtering](https://github.com/user-attachments/assets/b337654e-a465-45f3-82e8-49628bcf5023)